test/setup.js

```diff
@@ -47,4 +47,12 @@ global.navigator = {
 global.cancelAnimationFrame = function () {};
 global.requestAnimationFrame = function () {};
 window.MutationObserver = MutationObserver;
+
+class ResizeObserver {
```

Comment:
Måtte mocke denne for å få testen til å kjøre (pga tooltip på fjern-knappen)

Reply:
<img src="https://media2.giphy.com/media/YQitE4YNQNahy/giphy-downsized-medium.gif"/>